includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -759,25 +761,20 @@ class="stripe-btn-disabled give-hidden"
 									$account_name       = $details['account_name'];
 									$account_email      = $details['account_email'];
 									$stripe_account_id  = $details['account_id'];
-									$disconnect_message = ( 'connect' === $details['type'] ) ?
-										sprintf(
-											esc_html__( 'Are you sure you want to disconnect GiveWP from Stripe? If disconnected, this website and any others sharing the same Stripe account (%1$s) that are connected to GiveWP will need to reconnect in order to process payments.', 'give' ),
-											$stripe_account_id
-										) :
-										esc_html__( 'Are you sure you want to disconnect GiveWP from Stripe?', 'give' );
-									$disconnect_url     = ( 'connect' === $details['type'] ) ?
-										give_stripe_disconnect_url( $stripe_account_id, $slug ) :
-										add_query_arg(
-											[
-												'post_type' => 'give_forms',
-												'page'    => 'give-settings',
-												'tab'     => 'gateways',
-												'section' => 'stripe-settings',
-												'give_action' => 'disconnect_manual_stripe_account',
-											],
-											admin_url( 'edit.php' )
-										);
-
+									$disconnect_message = esc_html__( 'Are you sure you want to disconnect this Stripe account?', 'give' );
+									$disconnect_url     = add_query_arg(
+										[
+											'post_type'   => 'give_forms',
+											'page'        => 'give-settings',
+											'tab'         => 'gateways',
+											'section'     => 'stripe-settings',
+											'give_action' => ( 'connect' === $details['type'] )
+												? 'disconnect_connected_stripe_accout'
```

Comment:
@kjohnson Typo in string. Check suffix.

Reply:
Good catch.

Fixed by commit 9b88f1c2bcf5c17e0d2974768f5da81f5b57815d